Update org permissions matrix (#3366)
neondatabase/website: content/docs/manage/organizations.md

| [Invite/remove project collaborators](/docs/guides/project-collaboration-guide#invite-collaborators)                         |  ✅   |   ✅   |      ✅      |
| **Integration management**                                                                                                   |       |        |              |
| [Install GitHub integration](/docs/guides/neon-github-integration#install-the-github-app-and-connect-your-neon-project)      |  ✅   |   ❌   |      ❌      |
| [Install Neon Auth](/docs/guides/neon-auth#permissions)                                                                      |  ✅   |   ❌   |      ❌      |
| [Install the Neon Postgres Previews Integration on Vercel\*](/docs/guides/vercel-previews-integration#how-to-install)          |  ✅   |   ❌   |      ❌      |
| [Connect project to GitHub integration](/docs/guides/neon-github-integration#connect-more-neon-projects-with-the-github-app) |  ✅   |   ✅   |      ❌      |
| [Connect project (Neon Postgres Previews Integration)](/docs/guides/vercel-previews-integration#how-to-install)              |  ✅   |   ✅   |      ❌      |

> \* The Vercel Native Integrations are managed entirely in Vercel and use Vercel's permission system (see [Vercel Team Roles](https://vercel.com/docs/rbac/access-roles/team-level-roles)). For the Neon Postgres Previews Integration, projects must first be made available in Vercel before they can be connected to Neon.

Reviewer: ```suggestion
\* The Vercel Native Integration is managed entirely in Vercel and uses Vercel's permission system (see [Vercel Team Roles](https://vercel.com/docs/rbac/access-roles/team-level-roles)). For the Neon Postgres Previews Integration, projects must first be made available in Vercel before they can be connected to Neon.
```
